spec/controllers/cafe_works_controller_spec.rb

```diff
@@ -157,14 +157,14 @@
     context 'with no user' do
       it 'remove worker' do
         patch :remove_worker,
-              {
-                  id: cwork_access.to_param,
-                  cafe_work: {access_code: cwork_access.access_code}
-              }
-        cwork_access.reload
-
-        cwork_access.worker.present?.should be_falsey
-        response.should redirect_to(cwork_access)
+          {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.